server/controllers/uploadGeoJsonController.js

```diff
@@ -0,0 +1,19 @@
+import { GeoJSON } from "../models/geoJson.js";
```

Comment:
Users can upload GPX or Geojson, we need to support either both or convert to geojson then save